hc/api/tests/test_ping.py

```diff
@@ -62,12 +64,31 @@ def test_it_reads_forwarded_protocol(self):
         r = self.client.get("/ping/%s/" % self.check.code,
                             HTTP_X_FORWARDED_PROTO="https")
         ping = Ping.objects.latest("id")
-        ### Assert the expected response status code and ping's scheme
+        # Assert the expected response status code and ping's scheme
+        self.assertEqual(r.status_code, 200)
+        self.assertEqual(ping.scheme, "https")
 
     def test_it_never_caches(self):
         r = self.client.get("/ping/%s/" % self.check.code)
         assert "no-cache" in r.get("Cache-Control")
 
-    ### Test that when a ping is made a check with a paused status changes status
-    ### Test that a post to a ping works
-    ### Test that the csrf_client head works
+    # Test that when a ping is made a check with a paused status changes status
+    # Test that a post to a ping works
+    # Test that the csrf_client head works
+
+    def test_it_changes_status_of_paused_check(self):
+        # paused check
+        self.check.status = "paused"
+        self.check.save()
+        r = self.client.get("/ping/%s/" % self.check.code)
```

Comment:
Let's use new style string formatting https://www.python.org/dev/peps/pep-3101/
The % operator is deprecated in python 3.1 and will be removed in later versions. See the PEP about `format` instance method of template strings.